courses/machine_learning/deepdive2/structured/labs/5a_train_keras_ai_platform_babyweight.ipynb

```diff
@@ -0,0 +1,749 @@
+{
```

Comment:
<p>should include</p><p>os.environ["PROJECT"] = PROJECT</p><p>here or else later cells won't recognize the environment variable.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/674/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>export PROJECT=$(gcloud config list project --format "value(core.project)") already stores the project into an environment variable. Doing it again with os is redundant as pointed out in another one of my labs by Danielle.</p>

Reply:
<p>right. but later don't seem to recognize the PROJECT variable. I found I needed to set os.environ["PROJECT"] = PROJECT.</p><p>Can you please confirm it works as is?</p>

Reply:
<p>I added it back in.</p>